satpy/etc/composites/abi.yaml

```diff
@@ -1,6 +1,15 @@
 sensor_name: visir/abi
 
 modifiers:
+  # TODO: fix
```

Comment:
Remove `# TODO`. Also future note, usually a todo of fixing something is just `# FIXME: description` 😉